public/docs/advanced/configuration.md

```diff
@@ -79,6 +79,46 @@ For each network, if unspecified, transaction options will default to the follow
 * `from`: From address used during migrations. Defaults to the first available account provided by your Ethereum client.
 * `provider`: Default web3 provider using `host` and `port` options: `new Web3.providers.HttpProvider("http://<host>:<port>")`
 
+#### Accessing only one of multiple networks
```

Comment:
Note that this section only applies when using `provider` - using `host`/`port` instead means you don't have to do this function wrapping. Not sure if you want to mention that or not. Otherwise looks great!